src/Client/Campaign.php

```diff
@@ -26,6 +26,18 @@ public function getById(string $id): array
             throw $exception;
         }
 
-        return json_decode($response->getBody()->getContents(), true);
+        $campaign_as_json = json_decode($response->getBody()->getContents(), true);
```

Comment:
@alihejazi I would ideally make this camelCase to match all the other vars' style in the app. Otherwise I think this is good to go